### DISCUSSION TITLE
v0.6.5.1 Windows Installation issues

### DISCUSSION THREAD

**Discussion by RickTaylor79** [publicsamples/Quetzalcoatl#17] [General]
Ask questions about installing v0.6.5 on windows here. 

Before posting please  make sure you've followed [these instructions](https://github.com/publicsamples/Quetzalcoatl/wiki/Installation-v0.6.5).

If you are having problems with the expansion files, make sure you've deleted any previously installed content!

**Comment by convinced-prog** [comment]
One question before I go into hibernation.
Why is 'Q' still asking for Audio Resources.dat?
Thanks

**Comment by RickTaylor79** [comment]
Ha, I don't blame you after all this. Might join you. 

It shouldn't be asking for it as it's no longer required, which daw is this happening in? you might need to clear your plugin cache.

**Comment by convinced-prog** [comment]
Yes life is hard enough without having to fight your OS as well as other
application software.
All I want to do is have a bit of fun. i even managed to load it in LMMS
which is only VST2 via a plug-in called Element but still the same
nonsense. It's totally silent now. Of course??
Would a plugin rescan help? or would it just dredge up the same shit?
Yes an installer would be nice as I would be able to uninstall and start
afresh.
You know I'm starting to see double with this and I've developed a tic in
my left eye. Is that a bonus feature of 'Q'?
I must admit I still don't understand the /modularsamplea directory
structure. Your video isn't very clear and it has a virtual keyboard
slapped across everything. What exactly is it?
I have to laugh in the face of adversity.
Onward chaps!!
Thanks again

**Reply by convinced-prog** [reply]
OH NO NOT HIM AGAIN!!!
Cache clearance has done absolutely nothing
Res-can has done nothing
Still asking for that dat file, sometimes
Still demanding that buffer reset.
As a side. Iread on the HIME web-site that the developer is awareof this but refuses to do anything about it???
Still silent.
Can you describe the dir structure agian
And.......on my gravestone,simply say 'ASK Q'
Have a nice day

**Reply by RickTaylor79** [reply]
I'm at a loss. Can we:

1. Start from scratch. Delete everything related to Q from your system 
2. Pick a DAW to focus on, any is fine, but bouncing between different DAWs is confusing the hell out of me

**Comment by convinced-prog** [comment]
Right!
Do you want the good news or the not too bad news.
After bombing Cubase and totally locking it up, I settled on Reaper which is still the star DAW (so far)
I updatd my audio driver, ASIO4ALL v2
Buggered around with my midi keyboard drivers.
Restarted and Re-booted 5 zillion times
IT WORKED!
BUT! Still can't load Expansions or anything else that lives in the /Expansions dir. BTW What is this FACTORY thing I keep getting?
Still very erratic but getting there. Only works in Reaper as yet.
There you go!

**Reply by RickTaylor79** [reply]
That is good news! Good call on updating your driver too, the built in audio driver in windows is rubbish for real time audio.

Hmm, what do you see in the Expansion menu? Just FACTORY? Factory is the initial sound pack/presets (all the really interesting stuff is in the expansions).  If you're bored of the sounds there, take a look at [this video](https://www.youtube.com/watch?v=QzMKZoeWvho). You know you can load your own Wavs and SFZ?

I'll be posting (yet another) update shortly that should make expansions a bit simpler. Just click to install, no fussing with content folders.

**Reply by convinced-prog** [reply]
Just an update.
Had an audio meltdown which wasn't attributed to Q
This advice to everyone out there who is having commincation problems with midi keyboards or usb equipment, audio orientated.
Please disable any device listed in Windows Devices usb  tab that refers to  Intel SST USB. This is is a total disaster, download AASIO4ALL v2.16 which has a wrapper for this pernicious invention
Thanks to 'Q', in a sense, I have had my problems highlighted and hopefully resolved.
I spent a whole day fighting with my equipment until I updated my drivers and finally had everything restoed.
Anyway some people may already know about this but for those who don;t. There you go. Certainly restored my sanity.
Still usung Reaper. Cubase just doesn't like 'Q' or me, or anything else I throw at it. Way too complex for me
I just want to have fun, Not a brain clot.
Have a nice day!
